src/functions-and-arrays.js

```diff
@@ -1,41 +1,86 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(a, b) {
+  return a > b ? a : b;
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
-
+function findLongestWord(words) {
+  let longestLength = 0;
+  let longestWord = '';
+  if (words.length === 0) return null;
+  for (let word of words) {
+    if (word.length > longestLength) {
```

Comment:
I think it may be a bit redundant to save both the length and word of the longest word.

If you already have the longest word you can access its length at any time at `longestWord.length`